test/mailers/receipt_mailer_test.rb

```diff
@@ -1,27 +1,52 @@
 require 'test_helper'
 
 class ReceiptMailerTest < ActionMailer::TestCase
-  def test_receipt_email
-    thesis = theses(:one)
-    email = ReceiptMailer.receipt_email(thesis)
+  test 'sends confirmation emails' do
+    ClimateControl.modify DISABLE_ALL_EMAIL: 'false' do
+      thesis = theses(:one)
+      user = users(:admin)
+      email = ReceiptMailer.receipt_email(thesis, user)
 
-    # Send the email, then test that it got queued
-    assert_emails 1 do
-      email.deliver_now
-    end
+      # Send the email, then test that it got queued
+      assert_emails 1 do
+        email.deliver_now
+      end
```

Comment:
This block makes more sense to me, because we're calling a method on the email object - but on the previous file, we're patching a form route, not dealing with the email object.

Reply:
That's because this is where I copied that logic from! 😆